mod5.txt

```diff
@@ -0,0 +1 @@
+string: mongodb+srv://courtneyagregg:GodComplex11@mod5.nsqu7m2.mongodb.net/?retryWrites=true&w=majority&appName=Mod5
```

Comment:
Shouldn't be seeing this in the project itself! Make sure to have this excluded in your gitignore in the future.

Reply:
Will do!